.travis.yml

```diff
@@ -17,85 +12,76 @@ before_install:
 install:
   -
 
-env:
-  -JDK=Java8
-  -JDK=Java9
-  -JDK=Java10
-  -JDK=Java11-EA
-  -JDK=Java12-EA
-
 matrix:
   fast_finish: true
   include:
     - jdk: oraclejdk8
       env:
         - DESC="acceptance tests"
-        - JDK=Java8
         - CMD="mvn install --projects acceptance-tests --also-make --activate-profiles all"
 
     - jdk: oraclejdk8
       env:
         - DESC="findbugs"
-        - JDK=Java8
         - CMD="mvn install findbugs:check --projects '!scala-unit-tests,!jmh-scala-tests,!jmh-tests,!jcstress-tests,!p2-repository' --activate-profiles all -DskipTests=true"
+
     - jdk: oraclejdk8
       env:
         - DESC="checkstyle"
-        - JDK=Java8
         - CMD="mvn install checkstyle:check --activate-profiles all -DskipTests=true"
 
     - jdk: oraclejdk8
       env:
         - DESC="unit tests"
-        - JDK=Java8
         - CMD="mvn install"
 
     - jdk: oraclejdk8
       env:
         - DESC="compile performance-tests"
-        - JDK=Java8
         - CMD="mvn install --projects performance-tests --also-make --activate-profiles all -DskipTests=true"
 
     - jdk: oraclejdk9
       env:
-        - DESC="unit tests Java 9"
-        - JDK=Java9
+        - DESC="unit tests oraclejdk9"
         - CMD="mvn install"
 
-    - jdk: oraclejdk8
+    - jdk: openjdk10
+      env:
+      - DESC="unit tests openjdk10"
+      - CMD="mvn install"
+
+    - jdk: oraclejdk10
+      env:
+      - DESC="unit tests oraclejdk10"
+      - CMD="mvn install"
+
+    - jdk: openjdk11
       env:
-        - DESC="Java 10 regression"
-        - JDK=Java10
-        - CMD=""
+      - DESC="unit tests openjdk11"
+      - CMD="mvn install"
 
     - jdk: oraclejdk8
       env:
-        - DESC="Java 11 regression"
+        - DESC="unit tests Java11-EA"
         - JDK=Java11-EA
-        - CMD=""
+        - CMD="./.travis-command-ea-builds.sh"
 
     - jdk: oraclejdk8
       env:
-        - DESC="Java 12 regression"
+        - DESC="unit tests Java12-EA"
         - JDK=Java12-EA
-        - CMD=""
+        - CMD="./.travis-command-ea-builds.sh"
 
   allow_failures:
   - jdk: oraclejdk8
     env:
-      - DESC="Java 11 regression"
       - JDK=Java11-EA
-      - CMD=""
 
   - jdk: oraclejdk8
     env:
-      - DESC="Java 12 regression"
       - JDK=Java12-EA
-      - CMD=""
 
-script:
-  - ./.travis-command-ea-builds.sh
-  - eval travis_wait 55 $CMD
+script: eval travis_wait 30 $CMD
```

Comment:
This may have been broken before. travis_wait is a command that needs to wrap another command. The way it was written, I think the first command would run for up to 10 minutes, and then we'd wait up to 55 for a noop.